wordpress-mobile/WordPress-iOS
# Fix wp landing screen brush image stretching issue on iPad

Fixes #19484

This PR fixes an issue where the brush stroke background image on the WordPress Landing Screen appears stretched.

## Screenshots
| Before | After |
| ---- | --- |
| ![](https://user-images.githubusercontent.com/4588074/196684024-2878773e-100e-4959-b054-b6cf0f264c28.png) | ![iPad Pro - Light](https://user-images.githubusercontent.com/9609223/196842220-0640caa7-d8a3-45a5-b88d-fe759b93ffd8.png) |

| iPhone 14 Pro | iPhone SE |
| -------------- | ---------- |
| <img src="https://user-images.githubusercontent.com/9609223/196842470-2abb2d91-5b45-481e-94a6-0a8b0c1bb9be.png" width=414/>| <img src="https://user-images.githubusercontent.com/9609223/196842511-b968c709-323e-491c-a3ce-324a1e2e8ec6.png" width=320 /> |

## To test:
1. Open the WordPress app on an iPad
2. Enable the newWordPressLandingScreen feature flag
   - Log in
   -  Go to Me → AppSettings → 🐞 Debug
   - Enable newWordPressLandingScreen
3. Log out
4. Re-open the WordPress App
5. **Expect** the NEW landing screen to show
6. **Expect** The brush stroke background image should look like the attached screenshots or the [design](https://github.com/wordpress-mobile/WordPress-iOS/pull/19485#issuecomment-1284308455).

## Regression Notes
1. Potential unintended areas of impact
None.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
None.

3. What automated tests I added (or what prevented me from doing so)
None.

## PR submission checklist:
